defaults_settings.sh

```
@@ -0,0 +1,852 @@
+#!/bin/bash
+
+# macOS defaults settings exported on Wed May  7 16:32:47 PDT 2025
+# Run this script on a new machine to apply your preferred settings
+
+
+# com.apple.dock settings
+
+defaults write "com.apple.dock" "autohide" 1
+defaults write "com.apple.dock" "expose-group-apps" 0
+defaults write "com.apple.dock" "last-analytics-stamp" (
```

Shellcheck (0.10.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)